css/main.css

```diff
@@ -0,0 +1,396 @@
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.blue-color,
+.blue-color * {
+  color: #091e42 !important;
+}
+
+.ml-0 {
+  margin-left: 0 !important;
+}
+
+.card {
+  padding: 20%;
+  position: relative;
+  width: 80vw;
+  background-size: cover;
+  background-position: 50%;
+  text-shadow: 0 0 2px rgb(0, 0, 0);
+}
+
+.card-link {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+li {
+  list-style: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.input {
+  display: block;
+  border: 0;
+  border-bottom: 1px solid #dbd8d7;
+  width: 90vw;
+  margin-top: 15px;
+  height: 40px;
+  padding: 0 10px;
+}
```

Comment:
- [ ] You've done so well so far with the layout of the form and I am very much impressed but I would suggest that pay keen attention to the feedback of the previous reviewer about the danger of horizontal scroll in web design. It would be a very good idea to avoid the use of the ```vw``` unit of the width and replace the value with a percentage in this particular scenario. A possible code snippet may look something like this:
```
.class {
   width: 70%;
   margin-left: auto;
   margin-right: auto;
}
```